catalogue/aarch64-VMSA/tests/CASAL+FAIL+FH.litmus

```diff
@@ -0,0 +1,18 @@
+AArch64 CASAL+FAIL+FH
+Variant=handled

```

Comment:
Can we remove the handled variant? Also add vmsa.

Reply:
Just marking this unresolved to bring it back to your attention. I don't mind as much the test not having the variant vmsa but now handed is going to apply only for faults in P0 which I think is not the intention is it?

Reply:
Ah, I previously updated the wrong commit. Updated tests are now a part of the first commit too.